src/scheduled_tasks.rs

```diff
@@ -120,3 +133,80 @@ fn drop_ccnew_indexes(conn: &mut PgConnection) {
     .execute(conn)
     .expect("drop ccnew indexes");
 }
+
+// You can't use the structs from node_info.rs,
+// because many fields are missing from other platforms.
```

Comment:
I dont like this, better to mark fields as optional and use `#[serde(default)] ` attribute in the original structs. Then you can also store user numbers etc if they are present.

Reply:
K done.